polling_stations/apps/addressbase/management/commands/create_uprn_council_lookup.py

```diff
@@ -0,0 +1,76 @@
+from django.core.management.base import BaseCommand
+from django.db import connection
+from pathlib import Path
+
+
+class Command(BaseCommand):
+    """
+    This creates a lookup csv of uprn and council GSS codes.
+    This can then be imported using 'import_uprn_council_lookup'.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-d", "--destination", help="Path to write csv to", default=None
+        )
+
+    def handle(self, *args, **kwargs):
+
+        self.cursor = connection.cursor()  # Does this need to specify DEFAULT?
+        # Set where we'll write the join query to.
+        if kwargs["destination"]:
+            self.dst = Path(kwargs["destination"]).open("w+b")
+        else:
+            self.dst = Path("./uprn-to-councils.csv").open("w+b")
```

Comment:
Any particular reason we need to use binary mode here?

Reply:
I feel like it stopped something breaking, but I can't remember what and it seems to work fine with just `"w"`.